letsencrypt-auto

```diff
@@ -14,7 +14,19 @@ VENV_PATH=${VENV_PATH:-"$XDG_DATA_HOME/$VENV_NAME"}
 VENV_BIN=${VENV_PATH}/bin
 
 if test "`id -u`" -ne "0" ; then
-  SUDO=sudo
+  if type sudo 1>/dev/null 2>&1; then
```

Comment:
`type` doesn't seem to be part of the [POSIX shell standard](http://pubs.opengroup.org/onlinepubs/009695399/utilities/xcu_chap02.html), though it's supported in a lot of shells.